### PR TITLE
removed references to ::shadow due to its deprecation

### PR DESCRIPTION
**Pull request by gingerbeardman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,21 +9,21 @@ line number is marked.  Therefore, to modify the style of the line marker, you c
 like this in your styles.less file:
 
 ```sass
-atom-text-editor::shadow .line.folded {
+atom-text-editor .line.folded {
```
Comment (alysov) [inline README.md]: As per @vforge's comment in #6, should the first part of the selector have an `.editor` class as well:
`atom-text-editor.editor .line.folded {` ?

**Review by alysov — COMMENTED** [review]
Sorry for the delay.  GitHub's new review process is a bit misleading.

File: README.md (hunk 1 above)
Comment (gingerbeardman) [reply README.md]: you're right, it should.

**Comment by gingerbeardman** [comment]
Added .editor as required (untested)

File: README.md (hunk 1 above)
Comment (vforge) [reply README.md]: That was auto generated by the Atom, I did not verified it.

**Pull request merged**